pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/ddp/participant/ParticipantDto.java

```diff
@@ -101,6 +109,10 @@ public Builder(int ddpInstanceId, long lastChanged) {
             this.lastChanged = lastChanged;
         }
 
+        public Builder(int ddpInstanceId) {
```

Comment:
All callers use the current time anyway, so at some point we can eliminate the other constructor.